docs/Project-Configuration/Evergreen-Data-for-Analytics.md

```diff
@@ -58,6 +58,151 @@ Daily aggregated statistics for test executions run in Evergreen. Test stats are
 | total\_pass\_duration\_ns | DOUBLE  | Total duration, in nanoseconds, of passing tests.
 | task\_create\_iso         | VARCHAR | Date, in ISO format `YYYY-MM-DD`, on which the tests ran.
 
+### Evergreen Finished Tasks
+Finished Evergreen tasks, partitioned by the project and date (in ISO format). For example, the partition `mongodb-mongo-master/2022-12-05` would contain all tasks that completed on `2022-12-05` in the `mongodb-mongo-master` project. When running queries against this view it is highly recommended to always filter by project and date.
+
+#### Table
+| Catalog        | Schema          | View                                    |
+| ---------------|-----------------|-----------------------------------------|
+| awsdatacatalog | dev\_prod\_live | v\_\_evergreen\_\_finished\_tasks\_\_v0 |
+
+#### Columns
+| Name                           | Type      | Description |
+|--------------------------------|-----------|-------------|
+| task\_id                       | VARCHAR   | Unique task identifier.
+| execution                      | BIGINT    | Task execution number.
+| display\_name                  | VARCHAR   | Display name of the task.
+| version                        | VARCHAR   | ID of the version containing the task.
+| build\_id                      | VARCHAR   | ID of the build containing the task.
+| order                          | BIGINT    | Order number of the task's version. For patches this is the user's current patch submission count. For mainline versions this is the number of versions for that repository so far.
+| gitspec                        | VARCHAR   | Git commit SHA.
+| r                              | VARCHAR   | Requester type. Will always be one of: `patch_request`, `github_pull_request`, `gitter_request` (mainline), `trigger_request`, `merge_test` (commit queue), or `ad_hoc` (periodic build).
+| priority                       | BIGINT    | Scheduling priority of the task.
+| task\_group                    | VARCHAR   | Name of the task group that contains this task.
+| task\_group_max_hosts          | BIGINT    | Maximum number of hosts that will be used to run the task group.
+| task\_group_order              | BIGINT    | Position of this task in its task group.
+| depends\_on                    | ROW       | depends_on row describing the task's dependencies.
```

Comment:
This is an array:
```
ARRAY(ROW)
```
You can then just update the description to state `Array of depends_on rows describing...`